### ISSUE TITLE
Firefox UI: Save Draft button is broken for other languages

### ISSUE THREAD

**Issue by Mayank-Tripathi32** [WordPress/gutenberg#68065]
### Description

Save Draft text is hidden behind the header panel (Post Title), It seems to be related to fix in #67069 which fixed the issue previously for english language.

### Step-by-step reproduction instructions

1. Change your website language to `Deutsch (Österreich)` and go to create new post.
2. Reduce browser width to around 1200px wide
3. See Save Draft text (dutch version) appearing behind the document bar as per screenshot.

### Screenshots, screen recording, code snippet

![Image](https://github.com/user-attachments/assets/41d9dd21-c2ed-4f0a-b22d-dffe5aa29889)
![Image](https://github.com/user-attachments/assets/4ef9645f-6433-42d7-8899-a63a43a04079)

### Environment info

1. WP 6.7
2. Firefox 132.0.2
3. macOS Sonoma 14.6.1
4. No plugins
5. Twenty Twenty-Five
6. Gutenberg plugin: Disabled

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [x] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure

**Comment by ramonjd** [comment]
It's more noticeable in German, but seems to be rather an issue with the grid resizing in Firefox.

I'm not sure but it might be related to this CSS rule:

https://github.com/WordPress/gutenberg/blob/c5e8fd268328437220798137805c2fb623b25ed1/packages/editor/src/components/header/style.scss#L10-L10

**Comment by Mayank-Tripathi32** [comment]
@ramonjd 

```css

#1
 grid-template: auto / $header-height max-content minmax(50px, 1fr) max-content $header-height;

#2
 grid-template: auto / $header-height auto 2fr auto $header-height;

```

Tried following but only issue is that its not centered afterwards. Do you have any idea?

## Before
![Image](https://github.com/user-attachments/assets/9a7b95fe-9f6e-4179-ad61-414685af66f3)

## After
![Image](https://github.com/user-attachments/assets/58c1d92b-5d4f-4b11-98e1-497cb94cde3d)

**Comment by ramonjd** [comment]
Sorry, I don't have time to debug right now. I just took a guess, it might be something completely different, but there might be some hints out there on the web in relation to how Firefox deals with grid layout differently (if at all).  

Does it also occur on Safari or other browsers?

**Comment by Mayank-Tripathi32** [comment]
> Does it also occur on Safari or other browsers?

Nope, I actually had done a temp fix related to new icon previously.

**Comment by ellatrix** [comment]
Sounds like it should become horizontally scrollable?

**Comment by Mayank-Tripathi32** [comment]
> Sounds like it should become horizontally scrollable?

That or it should behave like it does on chrome and shrink for middle block.

**Comment by placoderm** [comment]
> Sounds like it should become horizontally scrollable?

No, that would work very poorly.



> That or it should behave like it does on chrome and shrink for middle block.

Yes, the middle part should shrink. There is more than enough space.

Also, this has nothing to do with "other languages". I am using English and have the same problem. I experience it specifically when using the PublishPress Checklist plugin that adds a new button next to the Publish button.

**Comment by Mayank-Tripathi32** [comment]
> > Sounds like it should become horizontally scrollable?
> 
> No, that would work very poorly.
> 
> > That or it should behave like it does on chrome and shrink for middle block.
> 
> Yes, the middle part should shrink. There is more than enough space.
> 
> Also, this has nothing to do with "other languages". I am using English and have the same problem. I experience it specifically when using the PublishPress Checklist plugin that adds a new button next to the Publish button.

~~This should actually be resolved with new version of gutenberg, as it removed the center block view as default.~~ Nevermind. Its still a problem for firefox users.

**Comment by annezazu** [comment]
Hey folks! I just tested this in Firefox using WP 6.8.3 using German (as that was mentioned above) and I can no longer replicate. Here's a video:

https://github.com/user-attachments/assets/fd02bc82-c99c-453b-a1bc-10fd87c2d2f0

I'm going to close this out as a result but **if this is wrong say so.** Happy to re-open and we definitely should get this fixed if it is still a problem.

**Comment by Mayank-Tripathi32** [comment]
Looks good on my end aswell. Thanks for checking 👍🏻
